client/landing/subscriptions/components/site-list/site-list.tsx

```diff
@@ -7,7 +7,9 @@ type SiteListProps = {
 	sites?: SiteSubscription[];
 };
 
-export default function SiteList( { sites }: SiteListProps ) {
+const defaultSites: SiteSubscription[] = [];
+
+export default function SiteList( { sites = defaultSites }: SiteListProps ) {
```

Comment:
This could be replaced by:
```suggestion
export default function SiteList( { sites = [] }: SiteListProps ) {
```

Reply:
When providing a default empty array value for a React component prop, it's recommended to use a `const` to avoid unnecessary re-renders.